### ISSUE TITLE
params.something 

### ISSUE THREAD

**Issue by s-aguinaga** [digitalcraftsman/hugo-strata-theme#10]
Great work on this theme. 
Is there a way to add arbitrary sections like the "about" and the "gallery"?  would I have to defined that in the `partials` folder?

**Comment by digitalcraftsman** [comment]
Hello @s-aguinaga,

your observation is right. All "sections" of the theme are defined as partials in the same-named folder. Create a new partial, adapt the html structure of existing ones (if you like) and reference you new partial in the index.html template.

**Comment by digitalcraftsman** [comment]
Take a look at the template of the [about section](https://github.com/digitalcraftsman/hugo-strata-theme/blob/master/layouts/partials/about.html).

Reopen this issue if you still have questions.

**Comment by MoreThanLuck** [comment]
Hey there, I'm a little confused on how this works. I'd like to make a site that's just a handful of `about` sections, then the `contact` form. I'm a multi-disciplinarian, so each section would just be a header and body for the different medium I do, like writing, photography, etc.

I made a few new html files in the partials folder, titling each for each section. So now there's a `writing.html`. `photography.html`, `filmmaking.html`, etc.

Given the `about` section is my template, I copied the contents into each new html file, except replacing the `about` in `.Site.Params.about.title` and `.Site.Params.about.content` with the name of the file, so `.Site.Params.writing.title`, etc.

Then, I tried to call them in the `config.toml` file with code like `[params.writing]`. However, nothing displays. 

What am I doing wrong?

Thanks!
